mocha_test/retryable.js

```diff
@@ -21,6 +21,30 @@ describe('retryable', function () {
         }, 15);
     });
 
+    it('basics with error test function', function (done) {
+        var calls = 0;
+        var special = 'special';
+        var opts = {
+            errorFilter: function(err) {
+                return err == special;
+            }
+        };
+        var retryableTask = async.retryable(opts, function (arg, cb) {
+            calls++;
+            expect(arg).to.equal(42);
+            cb(calls === 3 ? 'fail' : special);
+        });
+
+        retryableTask(42, function (err) {
+            expect(err).to.equal('fail');
+            expect(calls).to.equal(3);
+            done();
+        });
+
+        setTimeout(function () {
+        }, 15);
```

Comment:
I had used an existing test which had it; and I didn't notice it. Not sure why it's there to be honest. In the latest commit I removed it from the existing test and the new one. Tests pass locally for me, and in CI it seems.